### PR TITLE
Upgraded to the latest version of Play!

### PR DESCRIPTION
**Pull request by schmitch**
Changed the PlayVersion inside the build.sbt file.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -18,4 +18,4 @@ playBuildRepoName in ThisBuild := "play-mailer"
 
 mimaDefaultSettings
 
-previousArtifact := Some("com.typesafe.play" % "play-mailer_2.11" % "3.0.1")
\ No newline at end of file
+previousArtifact := Some("com.typesafe.play" % "play-mailer_2.11" % "3.0.1")
```
Comment (Mogztter) [inline build.sbt]: Could you revert this change (remove the newline at end of file)

Comment (schmitch) [inline build.sbt]: Yeah sorry, was my editors fault :(

**Comment by Mogztter** [comment]
Perfect, thanks!
